rust/Cargo.toml

```diff
@@ -5,13 +5,10 @@ cargo-features = ["edition2021"]
 members = [
     "abacus-core",
     "abacus-base",
+    "abacus-cli",
     "abacus-test",
     "chains/abacus-ethereum",
     "agents/kathy",
-    "agents/checkpointer",
     "agents/validator",
     "agents/relayer",
-    "tools/kms-cli",
```

Comment:
Perhaps kms-cli is included because maybe it'd be useful to humans in some debugging workflows, and those workflows have never been documented in a markdown file or something? I could believe that. But I could also believe the key management story has changed since the CLI was originally introduced and that it wouldn't even work anymore. Like you though, I haven't dug around to be fully certain -- hoping Nam knows off the top of his head.

Reply:
I think in the Optics context, something like kms-cli might have been useful to sign things manually with permissioned keys. I support deleting them for now. I suspect abacus-cli as an interface to read things from the DB may be useful and can be kept. Balance exporter probably can be removed, @mattiecnvr i believe you exporting them as prom metrics basically is strictly superior

Reply:
That makes sense, thanks for the reviews. This PR keeps abacus-cli, just moves it up one level in the directory tree since it's the only tool.